src/main/java/com/foundation/search/view/ParametersPanel.java

```diff
@@ -156,7 +340,84 @@ public JTextField getNameInput(){
      *
      * @return extensionInput
      */
-    public JTextField getExtensionInput(){
-        return extensionInput;
+    public String getExtensionInput() {
+        return extensionInput.getSelectedItem().toString();
+    }
+
+    /**
+     * This method returns an array of String with information to search by size
+     *
+     * @return sizeData
+     */
+    public String[] getSizeInput(){
+        sizeData = new String[2];
+        sizeData[0] = sizeComparison.getSelectedItem().toString();
+        sizeData[1] = sizeInput.getText();
+        sizeData[2] = sizeUnit.getSelectedItem().toString();
+        return sizeData;
     }
+
+    /**
+     * This method listens the actions in the checkboxes. overwrites the method
+     * existent in the ITemListener interface
```

Comment:
You can improve this description:
This method listens the performed actions on the checkboxes; it overwrites the existing method in the ItemListener interface

Reply:
done